components/molecules/RepoCardProfile/repo-card-profile.tsx

```diff
@@ -17,7 +17,7 @@ const RepoCardProfile = ({ avatar, orgName, repoName, prCount, issueCount, class
       <div className="h-max group-hover:border rounded-lg">
         <Avatar initials="BD" size={40} avatarURL={avatar} />
       </div>
-      <div className="flex flex-col gap-1 h-[46px]">
+      <div className="flex flex-col gap-1 h-[77px]">
```

Comment:
The problem is that we are using px and not the tailwind design system. This change fixes mobile but breaks desktop.

![Screen Shot 2023-08-15 at 4 00 39 PM](https://github.com/open-sauced/insights/assets/5713670/1bf2a01f-8d47-4210-b4df-e9200bd31c1f)